processors/pybossa_indications/processor.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from itertools import groupby
+
+import os
+import time
+import requests
+import pbclient as pbc
+import logging
+logger = logging.getLogger(__name__)
+
+PYBOSSA_URL = os.environ['PYBOSSA_URL']
+PYBOSSA_API_KEY = os.environ['PYBOSSA_API_KEY']
+project_id = 4228
```

Comment:
What if we have multiple projects? Should I create something like `PYBOSSA_INDICATIONS` and follow this convention for subsequent ones?